SurveyService/src/main/java/com/revature/controller/DistributionController.java

```diff
@@ -0,0 +1,43 @@
+package com.revature.controller;
+
+import java.io.File;
+import java.util.List;
+
+import java.util.Set;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * This controller takes in a batch id, or a batch id with a set of emails, then
```

Comment:
Place validation description in Service where it belongs.  Have this specifically explain purpose of rest endpoints.